storage/gc_queue_test.go

```diff
@@ -304,6 +305,13 @@ func TestGCQueueIntentResolution(t *testing.T) {
 	const now int64 = 48 * 60 * 60 * 1E9 // 2d past the epoch
 	tc.manualClock.Set(now)
 
+	// TODO: why is this needed to trigger intent resolution?
```

Comment:
ok. I filed #2497 for now, this PR's getting big enough. I'll take care of it after merge.